app/javascript/crayons/Modal/Modal.jsx

```diff
@@ -15,6 +15,7 @@ export const Modal = ({
   sheet,
   centered,
   noBackdrop,
+  showHeader = true,
```

Comment:
The `Modal` component is a bit opinionated on how the header should look, which differs from what we're trying to implement. This seemed like the most straightforward way to make this component a bit more flexible without going too wild...